common/src/main/java/net/whimxiqal/journey/search/function/CostFunctionType.java

```diff
@@ -28,5 +28,8 @@ public enum CostFunctionType {
   PLANAR_ORIENTED,
   MANHATTAN_DISTANCE,
   HEIGHT,
+
+  EUCLIDEAN_PLANAR_AVERAGE,
+
```

Comment:
Try to keep style consistent with surrounding code

Reply:
Fixed